web-app/js/portal/details/NcWmsPanel.js

```diff
@@ -383,6 +383,7 @@ Portal.details.NcWmsPanel = Ext.extend(Ext.Container, {
         this._updateTimeRangeLabel();
 
         this._setLayerSubsetExtent();
+        this._applyFilterValuesFromMap();
```

Comment:
I think this is a bit confusing.  Really, what we want to express at this level is `_applyFilterValuesToCollection`.  This method should in turn do the bit about getting the constraint from the map (it shouldn't really be getting it from *there*, but that's a different story).